src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java

```diff
@@ -93,19 +93,23 @@ public static void onEntityUpdate(LivingEvent.LivingUpdateEvent event) {
 //            }
 //        }
         List<EntityLivingBase> noGravityList = noGravityListMap.getOrDefault(event.getEntityLiving().getEntityWorld(), Lists.newArrayList());
-        if (eventEntityLiving.isPotionActive(RegistrarBloodMagic.SUSPENDED) && !eventEntityLiving.hasNoGravity()) {
-            eventEntityLiving.setNoGravity(true);
-            noGravityList.add(eventEntityLiving);
-        } else if (noGravityList.contains(eventEntityLiving)) {
-            eventEntityLiving.setNoGravity(false);
-            noGravityList.remove(eventEntityLiving);
-        }
+        if (noGravityList != null)
```

Comment:
No opening bracket for this case.

Reply:
No opening bracket needed, as there is only one statement below it (another conditional).

Reply:
No opening bracket needed, as there is only one statement below it (another conditional).

Reply:
A multi-line statement should still be wrapped for readability